pkgs/misc/themes/adapta/default.nix

```diff
@@ -34,9 +34,10 @@ stdenv.mkDerivation rec {
   postPatch = "patchShebangs .";
 
   configureFlags = [
-    "--disable-gtk_legacy"
-    "--disable-gtk_next"
-    "--disable-unity"
+    "--disable-gtk_next" # GTK 4.0
+    "--enable-gnome"
+    "--enable-parallel"
+    "--enable-plank"
```

Comment:
Would it make sense to add settings to en/disable those? And why are we not using the latest gtk?

Reply:
Well only reason I think we should go out of our way is for something like GTK 4.0...

Since it's not available yet! And AFAIK is still very much WIP and it's really really not worth building icons and putting them on your drive in anticipation or something.  So much so that I don't really think that anyone will find that a controversial stance (famous last words!) so LMK if that's not true please :).

I think the configure script disables that by default and I left it since it was there before and didn't want it to look like this may enable it.

The logic about enabling gnome was unclear so it seemed best to just ask for it.  Honestly I'm not sure I knew about plank but it's packaged in our repository so that's good enough for me in terms of "maybe someone'll want it".

I don't think it's worth the complexity to add flags here-- although maybe I'd feel differently if i looked at disk usage and it made a big difference.  All I know is in the last 24 hours i installed most of the icon/gtk themes in our repository (and they're still there so I can hop between while being picky and such) and if that sort of waste didn't motivate GC'ing or free'ing some roots I'm done with then I think we're good.

Very long answer short, I think this is good as-is and hopefully in the long reply you have an answer to your question about why we're not using GTK 4-- the key is it's "next" not "latest".

Reply:
Seems very reasonable. I usually prefer more configuration possibilities, but I see that it may not be worth the effort here. Didn't know about the GTK situation.